client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEditTable.tsx

```diff
@@ -177,6 +202,218 @@ export const OutboundLineEditTable = ({
     setVvmStatus,
   });
 
+  const packSize =
+    allocateIn.type === AllocateInType.Packs ? allocateIn.packSize : undefined;
+
+  const unit = Formatter.sentenceCase(item?.unitName ?? t('label.unit'));
+  const pluralisedUnitName = getPlural(unit, 2);
+
+  const mrtColumns = useMemo(() => {
+    const cols: ColumnDef<DraftStockOutLineFragment>[] = [
+      {
+        id: 'canAllocate',
+        header: '',
+        accessorFn: row => canAutoAllocate(row, packSize),
+        size: 0,
+        // TO-DO: Create "CheckboxCell" component
+        Cell: ({ renderedCellValue }) => {
+          return renderedCellValue ? '✓' : '';
+        },
+        defaultHideOnMobile: true,
+      },
+      {
+        accessorKey: 'batch',
+        header: t('label.batch'),
+        size: 60,
+        Cell: ({ row }) => {
+          return (
+            <div style={{ display: 'flex', alignItems: 'center' }}>
+              {row.original.batch}
+              {row.original.itemVariantId && (
+                <ItemVariantInfoIcon
+                  includeDoseColumns={allocateIn.type === AllocateInType.Doses}
+                  itemId={item?.id ?? ''}
+                  itemVariantId={row.original.itemVariantId}
+                />
+              )}
+            </div>
+          );
+        },
+      },
+      {
+        accessorKey: 'expiryDate',
+        header: t('label.expiry-date'),
+        size: 100,
+        columnType: ColumnType.Date,
```

Comment:
Our existing table, this is red when the stock is expired, and we show the row in grey, to help explain why it's not being autoallocated... can we replicate that here too? 🙏

Reply:
cb325688f6